app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -295,4 +295,10 @@
   <dimen name="topic_practice_footer_view_margin_top">24dp</dimen>
   <dimen name="topic_practice_footer_view_margin_bottom">0dp</dimen>
 
+  <!-- App Version -->
```

Comment:
You have deleted file from `res/layout-sw600dp` which is used for tablet (portait+landscape). But you have introduced these values only in `landscape-dimens` file. Please add same values in `values-sw600dp-port` too.

Reply:
Done